airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/array.sql

```diff
@@ -122,12 +130,16 @@ with numbers as (
 joined as (
     select
         _airbyte_{{ stream_name }}_hashid as _airbyte_hashid,
-        json_extract_array_element_text({{ column_col }}, numbers.generated_number::int - 1, true) as _airbyte_nested_data
+        {% if var("redshift_super_type", False) == True -%}
```

Comment:
this condition is used in several places. Here, [here](https://github.com/airbytehq/airbyte/pull/9610/files#diff-f99b9c631ef2f7e4985109ded6aecaf0eedd01f575a690fda74541f9af5ed85fR103) and [here](https://github.com/airbytehq/airbyte/pull/9610/files#diff-d62e4a8a19e6547243431d0d09aea84f7653ab6fe1947236e0d822cac1334eeaR12).Can we move it to a separate macros function?

Reply:
Good point, I have extracted to macros this part